azure-pipelines.yml

```diff
@@ -260,6 +261,7 @@ jobs:
         DISTRIB: 'conda'
         LOCK_FILE: './build_tools/azure/pylatest_conda_forge_mkl_osx-64_conda.lock'
         SKLEARN_TESTS_GLOBAL_RANDOM_SEED: '5'  # non-default seed
+        SCIPY_ARRAY_API=1
```

Comment:
If #29436 is merged first, a similar change will be needed for `pylatest_pip_openblas_pandas`.